python/GafferSceneUI/LightFilterUI.py

```diff
@@ -52,6 +52,18 @@
 
 	plugs = {
 
+		"filteredLights" : [
+
+			"description",
+			"""
+			The lights that are being filtered. Accepts a SetExpression. You
+			might want to set it to 'defaultLights' to have the filter affect
+			all lights that haven't been excluded from that set.
+			""",
+
+			"ui:scene:acceptsSetExpression", True,
```

Comment:
As mentioned in an earlier comment, I'm not sure this is helpful. Perhaps a preset for "defaultLights" might be a useful alternative? And what do you think to putting this in a "Light Linking" tab, analogous to the one on lights? I think it might be nice to separate it out from the regular parameters.

Reply:
I wouldn't mind the separated tab. I've put it right in the main one because we'd said that we'd like the plug to be easily discoverable as we don't link to anything by default. I'm happy to squirrel it away in its own tab, though :) I guess we need to discuss how much we consider the two mechanisms as separate things as per my previous comment before deciding what to call the tab?

Reply:
I almost think a separate tab draws more attention to it, rather than hiding it amongst the shader parameters. And yeah, it should be definitely considered alongside the StandardAttributes question.

Reply:
I've done it here: 5f4f1dd. Called it "Light Linking" for now as I have a feeling that's what we'll settle on. We can change it later if needed.